usagov/usagov-2021
# USAGOV-25-change_urls: Change the URL path of the Federal Directory

Change the path where the Federal Directory appears from /federal-agencies to /agency-index.

This is mostly configuration and documentation, but if you have already set up the Federal Directory, you'll need to go to your existing page at `/federal-agencies` and edit it, changing the URL alias to `/agency-index`

To test: 
1. Sync configuration
2. Update your existing `/federal-agencies` page as described above, or create a page with the URL alias `/agency-index` as described in docs/Federal_Directory.md
3. Rebuild caches
4. Add at least one Directory Record (if you haven't already) -- something that starts with "A." All it needs is the title (agency name). 
5. Verify that https://localhost/agency-index shows the directory. 


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-25

## Description


## Checklist for the Developer



## Checklist for the Peer Reviewers
